src/Symfony/Component/JsonPath/JsonCrawler.php

```diff
@@ -780,6 +782,10 @@ private function evaluateFunction(string $name, string $args, mixed $context): m
         $value = $argList[0] ?? null;
         $nodelistSize = $nodelistSizes[0] ?? 0;
 
+        if (\in_array($name, self::SINGULAR_ARGUMENT_FUNCTIONS, true) && $nodelistSize > 1) {
```

Comment:
```suggestion
        if ($nodelistSize > 1 && \in_array($name, self::SINGULAR_ARGUMENT_FUNCTIONS, true)) {
```
To make things a bit faster?

Reply:
Suits me fine 👍